ResourceSharing/WebApplication2/Migrations/ResourcesContextModelSnapshot.cs

```diff
@@ -0,0 +1,136 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+using WebApplication2;
+
+namespace WebApplication2.Migrations
+{
+    [DbContext(typeof(ResourcesContext))]
+    partial class ResourcesContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618

```

Comment:
Please add a comment on this line specifying the reason why these warnings are disabled

Reply:
It was generated automatically, so i have no idea, not under my jurisdiction